test/classes/ferguson/delete-free/owned-shared-fields.chpl

```diff
@@ -0,0 +1,109 @@
+use OwnedObject;
```

Comment:
Would be nice to add a comment explaining what this test is testing / why does it exist.